#! /bin/sh /usr/share/dpatch/dpatch-run
## 567593_improve_info_and_faq.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix several grammars, typos, the bug-reporting address.
## DP:
## DP: <URL:http://bugs.debian.org/567593>
## DP: <URL:https://bugs.g10code.com/gnupg/issue1188>
## DP: <URL:http://cvs.gnupg.org/cgi-bin/viewcvs.cgi?rev=5257&root=GnuPG&view=rev>
## DP: <URL:http://cvs.gnupg.org/cgi-bin/viewcvs.cgi?rev=5259&root=GnuPG&view=rev>

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' trunk~/doc/faq.raw trunk/doc/faq.raw
--- trunk~/doc/faq.raw	2008-12-11 17:39:51.000000000 +0100
+++ trunk/doc/faq.raw	2010-03-21 14:54:22.631771089 +0100
@@ -1026,18 +1026,20 @@
     lists? Did you have a look at the bug list (you'll find a link to
     the list of reported bugs on the documentation page). If you're not
     sure about it being a bug, you can send mail to the gnupg-devel
-    list. Otherwise, use the GUUG bug tracking system 
-    [H a href=http://bugs.guug.de/Reporting.html]<http://bugs.guug.de/Reporting.html>[H /a].
+    list. Otherwise, use the bug tracking system 
+    [H a href=http://bugs.gnupg.org]<http://bugs.gnupg.org>[H /a].
 
 <Q> Why doesn't GnuPG support X.509 certificates?
 
-    GnuPG, first and foremost, is an implementation of the OpenPGP
+    GnuPG 1.x, first and foremost, is an implementation of the OpenPGP
     standard (RFC 2440), which is a competing infrastructure, different
     from X.509.
 
     They are both public-key cryptosystems, but how the public keys are
     actually handled is different.
 
+    Note that GnuPG version 2.x fully supports S/MIME and X.509.
+
 <Q> Why do national characters in my user ID look funny?
 
     According to OpenPGP, GnuPG encodes user ID strings (and other
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' trunk~/doc/gpg.texi.rej trunk/doc/gpg.texi.rej
--- trunk~/doc/gpg.texi.rej	1970-01-01 01:00:00.000000000 +0100
+++ trunk/doc/gpg.texi.rej	2010-03-21 14:54:34.256804659 +0100
@@ -0,0 +1,91 @@
+--- doc/gpg.texi	(Revision 5258)
++++ doc/gpg.texi	(Revision 5259)
+@@ -586,7 +586,6 @@
+ related tasks.  It expects the specification of a key on the command
+ line.
+ 
+-
+ @c ******** Begin Edit-key Options **********
+ @table @asis
+ 
+@@ -894,7 +893,7 @@
+ @opindex passwd
+ Change the passphrase of the secret key belonging to the certificate
+ specified as @var{user_id}.  This is a shortcut for the sub-command
+-@code{passwd} in the edit key menu.
++@code{passwd} of the edit key menu.
+ @end ifclear
+ 
+ @end table
+@@ -909,7 +908,7 @@
+ @node GPG Options
+ @section Option Summary
+ 
+-@command{@gpgname} comes features a bunch of options to control the exact
++@command{@gpgname} features a bunch of options to control the exact
+ behaviour and to change the default configuration.
+ 
+ @menu
+@@ -1530,10 +1529,12 @@
+ ldaps).  Defaults to on.
+ 
+ @item ca-cert-file
+-Provide a certificate file to override the system default.  Only
++Provide a certificate store to override the system default.  Only
+ necessary if check-cert is enabled, and the keyserver is using a
+ certificate that is not present in a system default certificate list.
+ 
++Note that depending on the SSL library that the keyserver helper is
++built with, this may actually be a directory or a file.
+ @end table
+ 
+ @item --completes-needed @code{n}
+@@ -1767,16 +1768,26 @@
+ @option{--default-key}.
+ 
+ @item --try-all-secrets
++@opindex try-all-secrets
+ Don't look at the key ID as stored in the message but try all secret
+ keys in turn to find the right decryption key. This option forces the
+ behaviour as used by anonymous recipients (created by using
+ @option{--throw-keyids}) and might come handy in case where an encrypted
+ message contains a bogus key ID.
+ 
++@item --skip-hidden-recipients
++@itemx --no-skip-hidden-recipients
++@opindex skip-hidden-recipients
++@opindex no-skip-hidden-recipients
++During decryption skip all anonymous recipients.  This option helps in
++the case that people use the hidden recipients feature to hide there
++own encrypt-to key from others.  If oneself has many secret keys this
++may lead to a major annoyance because all keys are tried in turn to
++decrypt soemthing which was not really intended for it.  The drawback
++of this option is that it is currently not possible to decrypt a
++message which includes real anonymous recipients.
+ 
+ 
+-
+-
+ @end table
+ 
+ @c *******************************************
+@@ -2888,7 +2899,7 @@
+ @mansect bugs
+ @chapheading BUGS
+ 
+-On many systems this program should be installed as setuid(root). This
++On older systems this program should be installed as setuid(root). This
+ is necessary to lock memory pages. Locking memory pages prevents the
+ operating system from writing memory pages (which may contain
+ passphrases or other sensitive material) to disk. If you get no
+@@ -2903,6 +2914,10 @@
+ to protect the saved memory, passphrases or other sensitive material
+ may be recoverable from it later.
+ 
++Before you report a bug you should first search the mailing list
++archives for similar problems and second check whether such a bug has
++already been reported to our bug tracker at http://bugs.gnupg.org .
++
+ @mansect see also
+ @ifset isman
+ @command{gpgv}(1), 
